scipp_widgets/widgets.py

```diff
@@ -6,7 +6,7 @@
 import ipywidgets as widgets
 from .inputs import get_notebook_global_scope, IInput, Input
 from IPython.core.display import display, Javascript
-from typing import Any, MutableMapping, Callable, Iterable
+from typing import Callable, Iterable
 import scipp as sc
```

Comment:
Unrelated to this change: Maybe this import should be avoided here? Otherwise the package cannot be imported without scipp, even if no scipp-specific widget is used?

Reply:
Good point, is the best solution to separate widgets and inputs which do require scipp such as `PlotWidget` and `ScippObjectValidator` into a separate module so that the core module can be used without scipp?

Reply:
Not sure, maybe just import as needed, i.e., where scipp is actually used?

Reply:
Yes think that is a better idea.